src/windows/wslaservice/inc/wslaservice.idl

```diff
@@ -103,12 +110,89 @@ struct _VIRTUAL_MACHINE_SETTINGS {
 } VIRTUAL_MACHINE_SETTINGS;
 
 
-typedef
-struct _WSLA_SESSION_SETTINGS {
+
+struct WSLA_SESSION_SETTINGS {
     LPCWSTR DisplayName;
     // Details TBD.
-} WSLA_SESSION_SETTINGS;
+};
+
+
+struct WSLA_REGISTRY_AUTHENTICATION_INFORMATION
```

Comment:
What does this do?

Reply:
We can dig into this separately, but I think the SDK will be responsible for retrieving the right auth info, and just pass it as some sort of string blob to the host service.